src/Raven.Studio/typescript/viewmodels/database/settings/databaseSettings.ts

```diff
@@ -0,0 +1,444 @@
+
+        const dropDownCondition = this.groupToShow() === "allEntries" ||
+                                 (this.groupToShow() === "databaseEntries" && !entry.isServerWideOnlyEntry()) ||
+                                 (this.groupToShow() === "customizedDatabaseEntries" && entry instanceof models.databaseEntry && entry.override());
+
+        return categoryCondition && filterCondition && dropDownCondition;
+    }
+    
+    private entryContainsFilterText(entry: models.settingsEntry) {
+        const searchText = this.filterKeys().toLocaleLowerCase();
+        return entry.keyName().toLocaleLowerCase().includes(searchText);
+    }
+
+    canActivate(args: any) {
+        return $.when<any>(super.canActivate(args))
+            .then(() => {
+                const deferred = $.Deferred<canActivateResultDto>();
+
+                this.isForbidden(!accessManager.default.operatorAndAbove());
+
+                if (this.isForbidden()) {
+                    deferred.resolve({can: true});
+                } else {
+                    const db: database = this.activeDatabase();
+                    this.fetchDatabaseSettings(db)
+                        .done(() => deferred.resolve({can: true}))
+                        .fail((response: JQueryXHR) => {
+                            deferred.resolve({redirect: appUrl.forStatus(db)});
+                        });
+                }
+
+                return deferred;
+            });
+    }
+
+    compositionComplete() {
+        super.compositionComplete();
+        
+        this.initSummaryGrid();
+        
+        this.processAfterFetch();
+        
+        $('#database-settings-view [data-toggle="tooltip"]').tooltip();
+        
+        $('.description').on("mouseenter", event => {
+           const target = $(event.currentTarget);
+           const targetId = event.currentTarget.id;
+
+           if (!target.data('bs.popover')) {
+               popoverUtils.longWithHover(target, {
+                   content: `<div class="description-text"><strong>${targetId}</strong><small>${this.getDescriptionFor(targetId)}</small></div>`,
+                   placement: "bottom",
+                   html: true,
+                   container: "#database-settings-view"
+               })
+           }
+            
+           target.popover('show'); 
+        });
+    }
+    
+    private getDescriptionFor(keyName: string) {
+        const matchedEntry = this.allEntries().find(entry => entry.keyName() === keyName);
+        return matchedEntry.descriptionHtml();
+    }
+    
+    private initCategoryGrid() {
+        const categoriesGrid = this.categoriesGridController();
+        categoriesGrid.headerVisible(true);
+        categoriesGrid.init(() => this.fetchCategoriesData(), () =>
+            [
+                new hyperlinkColumn<string>(categoriesGrid, x => x, x => appUrl.forDatabaseSettings(this.activeDatabase()), "Category", "70%",
```

Comment:
why 70%?

Reply:
When using 100% then the horizontal scroll shows on the bottom.
It will Not show if for example 90% is used. (changed to 90% now)
![image](https://user-images.githubusercontent.com/17376974/81700357-3931c600-9471-11ea-917f-841bee3253db.png)

Reply:
ok